deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,364 @@
+    createdAt: '2020-07-01T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v0.7.0
+    certified: 'False'
+  name: opendatahub-operator.v0.7.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks.
+
+    ### Components
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.7.1 - Monitoring and alerting tool
+    * Grafana v6.2.4 - Data visualization and monitoring
+    * Airflow v1.10.2 - Workflow management
+    * AI-Library v0.6.0 - An open source collection of machine learning algorithms
+    * Seldon v1.1.0 - Open source platform for deploying machine learning models
+    * Argo v2.7.0 - Container-native Workflow Engine
+    * Apache Superset v0.34.0 - Open source application for data exploration and visualization
+    * Apache Kafka v2.4.0 - The open source stream processing platform
+
+    ### Available Channels
+
+    #### Beta
+
+    Channel `beta` offers the latest version of Open Data Hub operator. It is based on Kubeflow Operator and Kustomize for deployment configuration. This version is compatible with [Open Data Hub manifests](https://github.com/opendatahub-io/odh-manifests/) as well as [Kubeflow manifests](https://github.com/opendatahub-io/manifests). This channel will offer new components and features.
+
+    To deploy Kubeflow, use [this example](https://github.com/opendatahub-io/manifests/blob/v1.0-branch-openshift/kfdef/kfctl_openshift.yaml).
+
+    #### Legacy
+
+    Channel `legacy` offers Ansible Operator based version of Open Data Hub operator. It installs Open Data Hub [v0.5.2](https://gitlab.com/opendatahub/opendatahub-operator) which will only be updated with bug fixes, and will not receive new features and components.
+
+    To deploy `legacy` version, use [this example](https://gitlab.com/opendatahub/opendatahub-operator/-/blob/master/deploy/crds/opendatahub_v1alpha1_opendatahub_cr.yaml).
+
```

Comment:
this points to a kfctl with ```apiVersion: kfdef.apps.kubeflow.org/v1beta1``` I think it should be v1

Reply:
Submitted https://github.com/opendatahub-io/manifests/pull/47 to update the version